From 9d7c33c45d43290492b5ffd21dbee0442c764de8 Mon Sep 17 00:00:00 2001
From: Mike Purvis <mpurvis@clearpathrobotics.com>
Date: Sat, 7 Sep 2013 08:33:55 -0400
Subject: [PATCH 1/4] Added install rule for package.xml

---
 CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index b78708c..f2c961a 100755
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -68,3 +68,4 @@ add_subdirectory(src)
 add_subdirectory(docs)
 add_subdirectory(tests)
 
+install(FILES package.xml DESTINATION share/swiftnav)
-- 
1.9.1

