From b57b547e4ec25fca0523ce44751a88f067c1c97c Mon Sep 17 00:00:00 2001
From: Mike Purvis <mpurvis@clearpathrobotics.com>
Date: Thu, 3 Oct 2013 15:22:28 -0400
Subject: [PATCH 2/4] Avoid using architecture-specific optimizations in the
 build farm.

---
 CMakeLists.txt | 9 ---------
 1 file changed, 9 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index f2c961a..ea39cdf 100755
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -52,15 +52,6 @@ set(CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake")
 # Some compiler options used globally
 set(CMAKE_C_FLAGS "-Wall -Wextra -Wno-strict-prototypes -Wno-unknown-warning-option -Werror -std=gnu99 ${CMAKE_C_FLAGS}")
 
-if (NOT CMAKE_CROSSCOMPILING)
-  # Detect and use optimised compiler flags for the host architecture,
-  # this is specific to x86 family CPUs.
-  include(cmake/OptimizeForArchitecture.cmake)
-  OptimizeForArchitecture()
-  message(STATUS "Adding architecture flags: ${Vc_ARCHITECTURE_FLAGS}")
-  set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${Vc_ARCHITECTURE_FLAGS}")
-endif ()
-
 add_subdirectory(clapack-3.2.1-CMAKE)
 add_subdirectory(CBLAS)
 add_subdirectory(lapacke)
-- 
1.9.1

