From 1d70cdee4fa427706353020065047cfc568fdc41 Mon Sep 17 00:00:00 2001
From: Mike Purvis <mpurvis@clearpathrobotics.com>
Date: Thu, 3 Oct 2013 15:22:28 -0400
Subject: [PATCH 2/3] Avoid using architecture-specific optimizations in the
 build farm.

---
 CMakeLists.txt | 9 ---------
 1 file changed, 9 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index d7b326e..8508f9e 100755
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -26,15 +26,6 @@ set(CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake")
 # Some compiler options used globally
 set(CMAKE_C_FLAGS "-Wall -Wextra -Werror -std=gnu99 ${CMAKE_C_FLAGS}")
 
-if (NOT CMAKE_CROSSCOMPILING)
-  # Detect and use optimised compiler flags for the host architecture,
-  # this is specific to x86 family CPUs.
-  include(cmake/OptimizeForArchitecture.cmake)
-  OptimizeForArchitecture()
-  message(STATUS "Adding architecture flags: ${Vc_ARCHITECTURE_FLAGS}")
-  set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -O3 ${Vc_ARCHITECTURE_FLAGS}")
-endif ()
-
 add_subdirectory(src)
 add_subdirectory(docs)
 add_subdirectory(tests)
-- 
1.8.1.2

