From d19c9bf1fb4699414239db2c108dde527a91d255 Mon Sep 17 00:00:00 2001
From: Mike Purvis <mpurvis@clearpathrobotics.com>
Date: Sat, 26 Apr 2014 14:45:25 -0400
Subject: [PATCH 4/4] Downgrade CMake version requirement to match ROS Hydro.

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 8fb307f..c5e6631 100755
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.8.9)
+cmake_minimum_required(VERSION 2.8.3)
 project(libswiftnav)
 
 # Setup flags for Code Coverage build mode
-- 
1.8.5.2 (Apple Git-48)

